CONTRIBUTING.md

```diff
@@ -64,20 +76,55 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Member and Collaborator
+Collaborators are expected to follow this policy and continue to send pull requests, go through proper review, and have other collaborators merge their pull requests.
+
+### Becoming a Contributor
+
+To begin contributing to working groups and initiatives, start participating!
+
+Attend the meetings of the working group or initiative, investigate issues tagged
+as `good first issue`, open issues and pull requests, or provide insight via GitHub.
+
+Individuals can open an issue asking to be an Observer of CommComm meetings. You
+can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142)!
```

Comment:
Same as above. I thought the Observer status was to be dropped?

Reply:
Good recommendation on sidestepping existing connotations. I think I already saw a comment as a result of confusion related to this. Also, +1 to "Guest"; very welcoming.

Now adopting `Guest` across the board.